jobs/director/spec

```diff
@@ -115,6 +115,12 @@ properties:
   director.max_tasks:
     description: Max number of tasks per each type to keep in disk
     default: 2000
+  director.tasks_retention_period:
+    description: the retention period for tasks and their log files (days)
+    default: ''
+  director.tasks_deployments_retention_period:
+    description: the retention period for tasks and their log files of specific deployments (days)
+    default: ''
```

Comment:
Could you add an example like: https://github.com/cloudfoundry/haproxy-boshrelease/blob/master/jobs/haproxy/spec#L106